clusterloader2/pkg/measurement/common/ooms_tracker.go

```diff
@@ -55,13 +56,14 @@ func createClusterOOMsTrackerMeasurement() measurement.Measurement {
 }
 
 type clusterOOMsTrackerMeasurement struct {
-	selector  *measurementutil.ObjectSelector
-	msgRegex  *regexp.Regexp
-	isRunning bool
-	startTime time.Time
-	stopCh    chan struct{}
-	oomsLock  sync.Mutex
-	ooms      []oomEvent
+	selector         *measurementutil.ObjectSelector
+	msgRegex         *regexp.Regexp
+	isRunning        bool
+	startTime        time.Time
+	stopCh           chan struct{}
+	processWhitelist map[string]bool
```

Comment:
nit: processWhitelisted would be even better for set

Reply:
Agreed to change the field name to `processIgnored` (and use verb `ignore` instead of `allow` throughout the PR) after offline discussion.